msteams-platform/tutorials/UgradeTeamsUsingYeoman.md

```diff
@@ -0,0 +1,25 @@
+# Upgrade teams using yeoman generator
+This article helps you to upgrade teams from current version to the latest version
+## Get current version of teams
+```
+ yo teams --version
+```
+## Update teams
+The yo command lists out various options ranging from creating project to upate generator , use the below command to select updtae generator
+```
+ yo
+```
+Use the arrow keys to choose Update Generators
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams/YoSelectUpdateGen.png)
+
+This should listdown all the generators available , you should select a specific generator by using space bar to select and deselect teams from the available options
+![image](https://github.com/divya-akula/msteams-docs/blob/master/msteams-platform/assets/images/Update-Teams//UseSpaceToSelectGenerators.png)
```

Comment:
Kindly check and correct file paths, as they should not be pointing to personal folders.